components/flow/flow-service/src/main/kotlin/net/corda/flow/fiber/factory/FlowFiberFactoryImpl.kt

```
@@ -87,6 +87,8 @@ class FlowFiberFactoryImpl @Activate constructor(
                 flowFiberExecutionContext.flowCheckpoint.suspendCount,
                 flowFiberExecutionContext.sandboxGroupContext.sandboxGroup.id
             )
+        } catch (e: InterruptedException) {
+            throw e
```

bubble this back to flow engine exception handling rather than continuing to deserialize fiber